src/main/resources/db/migration/postgres/V1_1__initial.sql

```diff
@@ -0,0 +1,84 @@
+create table plan
+(
+    id            uuid primary key,
+    reference     uuid                        not null,
+    created_at    timestamp(3) with time zone not null,
+    created_by    varchar(50)                 not null,
+    updated_at    timestamp(3) with time zone not null,
+    updated_by    varchar(50)                 not null,
+
+    prison_number varchar(10)                 not null,
+    type          smallint                    not null
+);
+
+create table objective
+(
+    id                     uuid primary key,
+    reference              uuid                        not null,
+    created_at             timestamp(3) with time zone not null,
+    created_by             varchar(50)                 not null,
+    updated_at             timestamp(3) with time zone not null,
```

Comment:
should updated_at and updated_by allow nulls?

Reply:
I figured they would default to created_at/created_by but honestly I don't remember what all the jpa stuff does by default